design.py

```diff
@@ -1,40 +1,94 @@
 import sys
-import json
-#prompt a user to create a recipe
-#Ask for Ingredients
-#Ask for Instructions
+
 #Save/Store it into dictionary
 def create():
-    ask_recipe = ''
-    ask_instruction = ''
-    ask_ingredients = ''
-    recipe_dict = dict()
-    #counter = str(1)
-    print("Create a recipe! Type 'exit' to leave")
-    for ask_recipe in range (1):
-        ask_recipe = input("Recipe Name: ")
-        recipe_dict.update(ask_recipe = 'Nothing Yet')
-        if ask_recipe == 'exit':
-            exit(1)
-
-    while ask_instruction != 'exit':
-        ask_instruction = input("Instructions: ")
-        if ask_instruction == 'exit':
-            while ask_ingredients != 'exit':
-                ask_ingredients = input("Ingredients: ")  
-                if ask_ingredients == 'exit':
-                    #recipe_dict.update(ask_recipe = ask_instruction)
-                    #recipe_dict.update(ask_recipe, ask_ingredients)
-                    for key, value in recipe_dict.items():
-                        print(key,':',value)
-                        exit(1)
-
-
-
-    print("Create a recipe! Enter '0' to exit")
-    print(ask_recipe)
-    print(ask_instruction)
-    print(ask_ingredients)
+    
+    recipe_name = ''
+    recipe_dict = {}
+
+    print("Create a recipe!")
+    recipe_name = input("Recipe Name: ")
+
+    ingredientList = []
+    ingredients = ''
+
+    while True:
+        ingredients = input("Ingredients: ")  
+        if ingredients == 'exit':
+            break
+        ingredientList.append(ingredients)
+
+
+    instructionList = []
+    instructions = ""
+
+    while True:
+        instructions = input("Instructions: ")
+        if instructions == 'exit':
+            break
+        instructionList.append(instructions)
+
+    class ingr_instr_list:
```

Comment:
Few things here:

* Classes are usually defined before your functions and not within a function. Sometimes classes even get their own separate file or library where we store them so we can use them all over.
* Classes represent an object. Every time you define a class you should be thinking about "What am I representing with this?"
* Because Classes represent an object we name them after that object. In this case, the object you're trying to represent is a recipe. So we would simply name this class 'recipe'.
* Remember classes are just a collection of data structures(If you need a refresher on what is a data structure just ask). All you're doing is putting things you know about already into one convenient package so you can manipulate them easier and clearer.